docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,389 @@
+      </div>
+      <div className="project-name h5">{project.label}</div>
+    </a>
+  );
+
+  return (
+    <div className="featured-projects">
+      <div className="project-cards-container card-grid card-grid-3xN">
+        <ProjectCard project={projects[currentIndex]} index={currentIndex} />
+        <ProjectCard
+          project={projects[currentIndex + 1]}
+          index={currentIndex + 1}
+        />
+        <ProjectCard
+          project={projects[currentIndex + 2]}
+          index={currentIndex + 2}
+        />
+      </div>
+      <div className="arrow-wrapper d-flex justify-content-center mt-16">
+        <button
+          className={`arrow-button prev ${
+            currentIndex > 0 ? "hover-color" : ""
+          }`}
+          onClick={handlePrev}
+        >
+          <img alt="left arrow" />
+        </button>
+        <button
+          className={`arrow-button next ${
+            currentIndex < projects.length - 3 ? "hover-color" : ""
+          }`}
+          onClick={handleNext}
+        >
+          <img alt="right arrow" />
+        </button>
+      </div>
+    </div>
+  );
+};
+
+const ArticleCard = ({ article }) => {
+  return (
+    <div className="article-card-container">
+      <div className="article-card-background" />
+      <a
+        className="article-card col p-8 float-up-on-hover"
+        target="_blank"
+        href={article.url}
+      >
+        <div className="time h4 normal mb-8">{translate(article.time)}</div>
+        <div className="h5 mb-4">{translate(article.title)}</div>
+        <div className="btn btn-primary btn-arrow-out">
+          {translate("Read More")}
+        </div>
+      </a>
+    </div>
+  );
+};
+
+export default function XrplOverview() {
+  const modalRef = useRef(null);
+
+  return (
+    <div className="landing page-tokenization">
+      <div className="position-relative"></div>
+      <div className="position-relative d-none-sm"></div>
+      <section className="container-new mt-16">
+        <div className="card-grid card-grid-2xN">
+          <div className="col">
+            <img className="tokenization-graphic mw-100 hide-md" />
+          </div>
+          <div className="col">
+            <div className="d-flex flex-column-reverse">
+              <img className="tokenization-graphic mw-100 show-md" />
+              <h2 className="h4 h2-sm mb-10">
+                {translate(
+                  "Work with a variety of tokens supported by the XRP Ledger."
+                )}
+              </h2>
+              <h6 className="eyebrow mb-3 text-large">
+                {translate("Tokenization")}
+              </h6>
+            </div>
+            <p className="mb-10">
+              {translate(
+                "Explore the possibilities of tokenization on the XRP Ledger, revolutionizing ownership, transactions, and value exchange with unparalleled efficiency and security."
+              )}
+            </p>
```

Comment:
Another warning the page is throwing right now _might_ have to do with this section.

> Warning: Expected server HTML to contain a matching \<div> in \<p>.
> ```div
> p
> div
> div
> div
> section
> div
> XrplOverview
> ```